src/recipe.js

```diff
@@ -6,6 +6,8 @@ class Recipe {
     this.instructions = recipe.instructions;
     this.tags = recipe.tags;
     this.ingredientsData = ingredientsData;
+    this.isFavorite = false;
```

Comment:
Really like that you added a boolean value for this!

Reply:
Agreed - this REALLY helped in the coding.